### PR TITLE
Feature/43 person attributes

### PR DESCRIPTION
**Pull request by TheWalkingLeek**
Are there any model specs needed here? There's no logic inside the model so I figured not but it feels wrong not to add any specs...
refs: #43

### CODE REVIEW

File: app/models/cvp/person.rb [hunk 1]
```diff
@@ -0,0 +1,66 @@
+# encoding: utf-8
+
+#  Copyright (c) 2012-2020, CVP Schweiz. This file is part of
+#  hitobito_cvp and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito_cvp.
```
Comment (psunix) [inline app/models/cvp/person.rb:6]: we decided to get rid of those headers :) so only frozen_string_literal: true
also encoding: utf-8 is not required anymore

File: app/models/cvp/person.rb [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+#  correspondance_language   :string(255)      default('de'), not null
+#  civil_status              :string(255)      default('single'), not null
+
+module Cvp::Person
+  extend ActiveSupport::Concern
+
+  included do
+    include I18nEnums
+
+    Person::PUBLIC_ATTRS << :title << :website << :correspondance_language << :civil_status
```
Comment (psunix) [inline app/models/cvp/person.rb:57]: Person::PUBLIC_ATTRS.push(:title, :website, ...) ?

**Review by psunix — APPROVED** [review]
:+1:

**Review by amaierhofer — COMMENTED** [review]
@TheWalkingLeek Steht nicht explizit im Ticket aber es macht natürlich noch Sinn, wenn diese Felder editiertbar und anzeigbar sind.

**Comment by amaierhofer** [comment]
@TheWalkingLeek @psunix 

Lizenz Headers sollen drinnen bleiben. Ist sonst nach was in Progress (wegen dem WIP) commit oder ist das jetzt good2go?

**Comment by TheWalkingLeek** [comment]
Bin noch an den Views, hab die aber bald fertig und ist dann ready. Melde mich wenns soweit ist

**Comment by TheWalkingLeek** [comment]
Ist jetzt good2go

**Review by psunix — CHANGES_REQUESTED** [review]
code looks good.
did you copy this with the Salutation pseudo model from somewhere?
I cannot save any of the new attributes like 'title', 'civil_status' ... changes are not persisted.
there's 0 tests ... maybe it makes sense to add at least one test that checks if the custom attributes are there and working probably.

**Comment by TheWalkingLeek** [comment]
The Salutation implementation is done according to pbs.
The saving #WorksOnMyMachine, I also added the new attributes to the permitted_params inside wagon.rb. Did you reboot the server? @psunix 
I'm gonna look into expanding the specs

**Pull request merged**